pkgs/development/python-modules/spacy/lookups-data.nix

```diff
@@ -0,0 +1,37 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, setuptools
+, spacy
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "spacy-lookups-data";
+  version = "1.0.3";
+  format = "setuptools";
+
+  src = fetchPypi {
+    pname = "spacy_lookups_data";
+    inherit version;
+    hash = "sha256-q2hlVI+4ZtR5CQ4xEIp+Je0ZKhH8sJiW5xFjKM3FK+E=";
+  };
+
+  propagatedBuildInputs = [
+    setuptools
```

Comment:
Is `setuptools` required for runtime?
I mean, in most cases, it is only needed at build time.

Reply:
Good point, I didn't check the output of `nix-init` closely enough.

Reply:
Since we use `format = "setuptools"`, we don't need setuptools in buildInputs (or nativeBuildInputs) either.

besides this, LGTM.